src/main/java/com/google/graphgeckos/dashboard/storage/Builder.java

```diff
@@ -16,37 +16,93 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Map;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
 import org.springframework.cloud.gcp.data.datastore.core.mapping.Entity;
+import org.springframework.data.annotation.Transient;
 
 /**
  * Contains the information retrieved from a single build bot. It is used as a member of BuildInfo...
- * of {@link #BuildInfo BuildInfo}, and for converting ParsedBuildbotData objects
- * to Builder objects, used by BuildInfo.
+ * of {@link BuildInfo}
  */
 @Entity(name = "builder")
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class Builder {
 
+  @Transient
+  private String commitHash;
+
+  @Transient
+  private String branch;
```

Comment:
If we do not want to store the branch, why do we need to take it at all?

Reply:
Because in Java it is not allowed to mark fields as final if they are not assigned in the constructor.

Reply:
Because it is used in the constructor of the BuildInfo class.